core/domain/story_services.py

```diff
@@ -970,19 +970,32 @@ def record_completed_node_in_story_context(
         progress_model.put()
 
 
-def get_chapter_notification_dicts() -> Dict[str, List[Dict[
-    str, Sequence[str]]]]:
-    """Returns a dict of behind schedule and upcoming chapters."""
+def get_chapter_notification_dicts() -> Tuple[
+    List[story_domain.OverdueStoryDict],
+    List[story_domain.UpcomingStoryDict]]:
```

Comment:
These should be fully-typed domain objects.

Reply:
Done